rendermodules/lens_correction/apply_lens_correction.py

```diff
@@ -75,6 +79,23 @@ def run(self):
         lc_tform.transformId = refId
         ref_lc = renderapi.transform.ReferenceTransform(
             refId=lc_tform.transformId)
+
+        tspecs = renderapi.tilespec.get_tile_specs_from_z(
+            self.input_stack, self.zValues[0], render=r)
+        levels = [int(l) for l in tspecs[0].ip.levels]
+        # make mask mipmaps
+        mask_mm_list = []
+        if self.args['maskUrl'] is not None:
+            root, ext = os.path.splitext(self.args['maskUrl'])
+            mask_mm_list = create_mipmaps(
```

Comment:
I think this defaults to using `skimage.measure.block_reduce` applying a `numpy.mean` function, which can lead to non-binary masked pixels.  A min, max, or median reduction function would probably work better for a mask.